internal/commands/list_registries.go

```diff
@@ -0,0 +1,32 @@
+package commands
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/buildpacks/pack/internal/config"
+	"github.com/buildpacks/pack/logging"
+)
+
+func ListRegistries(logger logging.Logger, cfg config.Config) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list-registries",
+		Args:  cobra.NoArgs,
+		Short: "Lists all buildpack registries",
+		RunE: logError(logger, func(cmd *cobra.Command, args []string) error {
+			for _, registry := range cfg.Registries {
+				if registry.Name == cfg.DefaultRegistryName {
+					logger.Infof("* %s", registry.Name)
+				} else {
+					logger.Infof("  %s", registry.Name)
+				}
+				if logger.IsVerbose() {
+					logger.Infof("    Type: %s", registry.Type)
+					logger.Infof("    URL:  %s", registry.URL)
+				}
```

Comment:
Would adding this as a debug comment work here? I think it should also only print if verbose, and simplify this a bit.

Reply:
@dfreilich I have a question.  Do you know if all the log functions e.g. `Info`, `Debug`, etc.  print to `stderr`?

Reply:
Only `Error` prints to stderr:

https://github.com/buildpacks/pack/blob/main/internal/logging/logging.go#L90-L92

Reply:
Ok, good to know.  Generally speaking, I try and only reserve parse-able output to `stdout`.  I don't know if this output would necessarily classify as parse-able, but at the same time, I don't want `Error:` showing-up in the log output.

Reply:
@dfreilich I realise this has been outdated, but do you still have any thoughts on the new changes?

Reply:
@jromero I do wonder if I should be outputting to `stderr` instead of `stdout`

Reply:
As a CLI, I don't understand why we would want to. What would we be trying to achieve by printing to stderr?

Reply:
@jromero best practices suggest reserving `stdout` for things that are easily parse-able.  This is especially helpful when our users write scripts that use `pack`.

As I think more about this particular case it's probably to write to `stdout`.

Reply:
I definitely agree that this should be printed to `stdout`, rather than `stderr`.